backend/btrixcloud/operator/crawls.py

```diff
@@ -1274,13 +1287,20 @@ async def update_crawl_state(
 
         # check if done / failed
         status_count: dict[str, int] = {}
-        for i in range(crawl.scale):
+        for i in range(status.scale):
             res = results.get(f"crawl-{crawl.id}-{i}")
             if res:
                 status_count[res] = status_count.get(res, 0) + 1
 
-        # check if all crawlers are done
-        if done and status_count.get("done", 0) >= crawl.scale:
+        num_done = status_count.get("done", 0)
+        num_failed = status_count.get("failed", 0)
+        # all expected pods are either done or failed
+        all_completed = (num_done + num_failed) >= status.scale
+
+        # if at least one is done accordion to redis, consider crawl successful
```

Comment:
```suggestion
        # if at least one is done according to redis, consider crawl successful
```

music on the mind? ;)

Reply:
yep 🎹